client/src/components/admin/AdminCompetitionEvents/AdminCompetitionEvents.jsx

```diff
@@ -21,6 +21,8 @@ const COMPETITION_EVENTS_QUERY = gql`
           name
           open
           finished
+          enteredResults
+          totalResults
```

Comment:
Reading this names without context one may assume the values is a list of results (technically we don't have a nested `{}`, but that's a detail). I would make the naming more explicit, as in `numEnteredResults` and `numResults`.